dataset/dictionary/characters.json5

```diff
@@ -2316,7 +2316,7 @@
   {
     en: "Dunyarzad Homayani",
     ja: "ドニアザード・フーマイ",
-    zhCN: "迪娜泽黛",
+    zhCN: "迪娜泽黛·呼玛伊",
```

Comment:
@Bill-Haku @SleepyAsh0191 Can you review if this translation is correct?

Reply:
yes that's correct

Reply:
Thanks!